index.html

```diff
@@ -311,7 +311,7 @@ <h1 id="contact-text">
         ></textarea>
         <button id="btn" type="submit">Get in touch</button>
```

Comment:
- [ ] The get in touch button is currently failing the contrast test, as it is a button for our form, I think it would be best if we fix the contrast for this button as it will definitely enhance the accessibility of our site.
current result :point_down: 
![image](https://user-images.githubusercontent.com/97521557/201992851-d549630f-b6ab-4de4-95a7-1cc175a81563.png)

you can check the contrast here https://www.aditus.io/button-contrast-checker/